Sources/MarathonCore/Command.swift

```diff
@@ -90,6 +93,8 @@ extension Command {
             return ""
         case .help:
             return ""
+        case .export:
+            return "<script-path> [<export-path>] [--force]"
```

Comment:
So should `marathon export --force myScript` be allowed? Running `marathon create --no-xcode myScript` creates a script called `--no-xcode`.